### PR TITLE
feat(stepper): add tooltip in stepper

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Add tooltip in stepper

## Why

Need for registration stepper in portal

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@catena-x/portal-shared-components",
-  "version": "2.1.11",
+  "name": "@nidhi.garg/portal-shared-components",
```
Comment (manojava-gk) [inline package.json:2]: pls change this back to catena-x

File: src/components/basic/Stepper/Stepper.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
```
Comment (manojava-gk) [inline src/components/basic/Stepper/Stepper.scss:2]: remove 1st line. i.e line number 2

**Review by manojava-gk — CHANGES_REQUESTED** [review]
findings

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@catena-x/portal-shared-components",
-  "version": "2.1.11",
+  "version": "2.1.14",
```
Comment (manojava-gk) [inline package.json:3]: why not 2.1.12?

**Review by manojava-gk — COMMENTED** [review]
Small clarification. Hope you have run all linter rules on this PR

File: package.json (hunk 3 above)
Comment (nidhigarg-bmw) [reply package.json]: Made the change. And yes I ran the linter command already.

**Review by oyo — APPROVED** [review]

**Pull request merged**